subnet/deployment-generator/src/faucet.js

```diff
@@ -0,0 +1,175 @@
+process.chdir(__dirname);
```

Comment:
It's better if you can separate `Transfer` and `Server` code into different file.

Reply:
Let me do this next update. There is a need to restructure the whole 'src' folder. Now there are 3 different projects (gen.js, ui.js, faucet.js), it gets really confusing.